src/Microsoft.DotNet.Cli.Utils/Telemetry.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
+using Microsoft.Extensions.PlatformAbstractions;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class Telemetry : ITelemetry
+    {
+        private static bool _isInitialized = false;
+        private static TelemetryClient _client = null;
+
+        private static Dictionary<string, string> _commonProperties = null;
+        private static Dictionary<string, double> _commonMeasurements = null;
```

Comment:
I am not seeing the point of having _isInitialized, _client, _commonProperties, and _commonMeasurements be static.  The appear to be initialized each time the ctor is called and only referenced by instance members.  Seems like this code would be holding references unnecessarily and prevent GC.